config.toml

```diff
@@ -7,3 +7,32 @@ keyframes = [
 	{ time = 0.5, rot = 190, bottom_padding = -10 },
 	{ time = 0.8, rot = 360, bottom_padding = 0 },
 ]
+
+tilemap = """\
+gr,gr,gr,gr,gr
+di,di,di,di,di
+gr,di,gr,di,gr
+di,gr,di,gr,di
+gr,gr,di,di,gr
+gr,gr,di,di,gr
+gr,gr,gr,gr,gr
+fr,fr,fr,fr,bd
```

Comment:
In most monospaced text renderers, the ratio of the length of a character to the height of a character is 1:2.
For this reason, if we have two letters per map tile, you can have a square for each tile, making the text tile map a scale model of the graphical one.